plugins/package-managers/gleam/src/main/kotlin/GleamToml.kt

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (C) 2025 The ORT Project Copyright Holders <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.gleam
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+import net.peanuuutz.tomlkt.Toml
+import net.peanuuutz.tomlkt.TomlElement
+import net.peanuuutz.tomlkt.TomlLiteral
+import net.peanuuutz.tomlkt.TomlTable
+import net.peanuuutz.tomlkt.asTomlLiteral
+import net.peanuuutz.tomlkt.asTomlTable
+
+internal val toml = Toml { ignoreUnknownKeys = true }
```

Comment:
As this is also used to deserialize `GleamManifest`, and only accessed from `Gleam.kt`, I believe it should rather be moved there.